packages/manager/modules/pci/src/projects/project/storages/databases/database/advanced-configuration/advanced-configuration.scss

```diff
@@ -0,0 +1,5 @@
+ovh-manager-pci-project-database-advanced-configuration {
+  .ui-select-dropdown {
```

Comment:
We tend to avoid adding custom styles.
**question:** Which issue are we trying to solve here? 
A `<oui-select>` inside a `<oui-inline-adder>`? If yes, it might be better to solve this issue directly in the component style.

Reply:
The problem is that the `sticky-top` helper we use on the left column sets a z-index of 1250.
When in mobile vue, the `oui-select `goes below this element (see screenshot)

![image](https://user-images.githubusercontent.com/24541481/157192336-0adee480-9ee0-42df-a932-708e769099bb.png)

I think this is a very niche problem, and I'm not sure it would not break in other places, that's why I isolated the css to this page.